### ISSUE TITLE
Make redirect_to_first_child() function

### ISSUE THREAD

**Issue by drewbaker** [funkhaus/style-guide#9]
We could make this nicer now:

```
        case is_page('film-tv') :
            // Redirect to first child page.
            $pagekids = get_pages("child_of=".$post->ID."&sort_column=menu_order");
            $firstchild = $pagekids[0];
            if( $firstchild ) {
                wp_redirect(get_permalink($firstchild->ID), 301);        
                exit;
            } else {
                get_template_part('index');
            }
            break;
```

**Comment by SaFrMo** [comment]
@drewbaker Added function in commit 59ce096 on branch `sander` - waiting on approval to bring into `master`

**Comment by SaFrMo** [comment]
Also added `return false;` as the default action if a first child is not found in commit a98146b on `sander`

**Comment by SaFrMo** [comment]
Fixed this issue in commit dd93d04 on branch `sander`
